data-prepper-plugins/mongodb/src/test/java/org/opensearch/dataprepper/plugins/mongo/stream/StreamWorkerTest.java

```diff
@@ -376,7 +376,7 @@ void test_processStream_terminateChangeStreamSuccess() {
         when(s3PartitionStatus.getPartitions()).thenReturn(partitions);
         when(mockPartitionCheckpoint.getGlobalS3FolderCreationStatus(collection)).thenReturn(Optional.of(s3PartitionStatus));
         Event event = mock(Event.class);
-        when(event.get("_id", Object.class)).thenReturn(UUID.randomUUID().toString());
+        //when(event.get("_id", Object.class)).thenReturn(UUID.randomUUID().toString());
```

Comment:
There are some extra comments in some of these tests.

Reply:
Will have follow up CR to enable this.